src/server/routes/index.js

```diff
@@ -4,29 +4,76 @@ const item = require('./item');
 const auth = require('./auth');
 const products = require('./products');
 
+/**
+ * Verifies if the user has an active session,
+ * it redirects him to the login router if the given data is incorrect.
+ * @param {object} req - Express object
+ * @param {object} res - Express object
+ * @param {object} next - Express object
+ * @return {object} Give the control to the next step in the middleware.
```

Comment:
this is not a return, is using the `res` and `next` as callback functions

Reply:
Fix commit 3cfeb5e92cfb34585517a7b26ccbab5f23a70d5f